src/test/java/deineka/BaseTest.java

```diff
@@ -0,0 +1,20 @@
+package deineka;
+
+import Utils.InstanceDriver;
+import Utils.UtilsHelper;
+import Utils.DriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+public class BaseTest {
+    @BeforeMethod
```

Comment:
add Lombok plugin into a project and install this plugin in intelijIdea for annotation @Getter

```
@Getter
    protected WebDriver driver;

    protected GoogleSearchPage googleSearchPage;
    protected FollowLinkPage followLinkPage;

    @BeforeMethod
    public final void setUp() {
        WebDriverManager.chromedriver().setup();
        driver = new ChromeDriver();

        driver.manage().window().maximize();

        driver.get(UtilsHelper.getProperty("page.url"));

        googleSearchPage = new GoogleSearchPage(driver);
    }

    @AfterMethod
    public final void tearDown() {
        driver.quit();
    }
```